app/views/static/about.html.erb

```diff
@@ -27,7 +27,7 @@
                     blurb_header_klass: " help-header",
                     blurb_header: t('static.about.help.blurb_header'),
                     blurb_body_klass: " help-blurb",
-                    blurb: t('static.about.help.blurb').html_safe
+                    blurb: "Please visit our <a href=\"https://blackcat-test.library.emory.edu/contact\">Contact page</a> for more information about Emory Libraries collections, catalog, electronic resources, and access. You can also reach out via our feedback form or visit our project wiki.".html_safe
```

Comment:
Have you tried putting this string into config/locales/en.yml instead of deleting it completely?